fuzzer/src/request_build_helper.py

```diff
@@ -34,10 +36,12 @@ def _is_header_in_config(headers, header_name):
         return headers is not None and header_name in headers
 
     @staticmethod
-    def generate_uri(uri, uri_parameters, config, fuzzable=False):
-        fixed_attributes = config["fixed_url_attributes"] if "fixed_url_attributes" in config else None
+    def generate_uri(uri, uri_parameters, fuzzable=False):
         id_generator = _unique_uri_attribute_id()
 
+        already_used_parameters: List[str] = []
+
+        # 1] Generate URI as it is in payloads file
```

Comment:
instead of comments you can create 2 self describing private functions that you can call here